std/bigint.d

```diff
@@ -1031,3 +1031,12 @@ unittest // 13391
     x2 /= 123456789123456789UL;
     assert(x2 == 1);
 }
+
+unittest // 13963
+{
+    BigInt x = 1;
+    assert(is(typeof(x % 1) == int));
+    assert(is(typeof(x % 1U) == int));
+    assert(is(typeof(x % 1L) == BigInt));
+    assert(is(typeof(x % 1UL) == BigInt));
```

Comment:
Can you also add some non-trivial checks to make sure it returns the right value as well?